src/tests/test_utils.py

```diff
@@ -0,0 +1,112 @@
+import time
+import unittest
+
+from tests.shared_test_setup import TestSetUp
+from tkpysdk.utils.network import Network
+from tkpysdk.utils.order_utils import get_order_history
+from tkpysdk.utils.position_utils import create_position, CreatePositionResponse, get_position_details, \
+    remove_liquidity_and_burn
+from tkpysdk.utils.quote_utils import get_erc20_balance, get_order_book
```

Comment:
the idea usage for all of these functions should be 
```
from tkpysdk import get_erc20_balance, get_order_book
```

this should work after _init_.py file has been setup properly by exporting all external functions directly